### PR TITLE
Added labels in marker creation dialog + added spacing in grid

### PR DESCRIPTION
**Pull request by SebastienForay**

### CODE REVIEW

**Review by cloudkucooland — APPROVED** [review]
looks good to me

File: src/code/css/wasabee.css [hunk 1]
```diff
@@ -589,14 +589,16 @@
 .wasabee-dialog-markeradd div.content {
   display: grid;
   grid-template-columns: repeat(2, auto);
+  row-gap: 4px;
+  column-gap: 4px;
```
Comment (le-jeu) [inline src/code/css/wasabee.css:593]: you can add `align-items: baseline;` to fix label/input vertical alignement

File: src/code/dialogs/markerAddDialog.js [hunk 2]
```diff
@@ -85,12 +85,18 @@ const MarkerAddDialog = WDialog.extend({
     const content = L.DomUtil.create("div", "content");
     this._portal = L.DomUtil.create("div", "portal", content);
 
+    L.DomUtil.create("label", null, content).textContent = "Type";
```
Comment (le-jeu) [inline src/code/dialogs/markerAddDialog.js:88]: todo wX strings

File: src/code/css/wasabee.css (hunk 1 above)
Comment (SebastienForay) [reply src/code/css/wasabee.css:593]: Thanks for the tip 👍

File: src/code/dialogs/markerAddDialog.js (hunk 2 above)
Comment (SebastienForay) [reply src/code/dialogs/markerAddDialog.js:88]: Right, I'll do it and add missing translations in the dialog

**Review by le-jeu — APPROVED** [review]

**Pull request merged**